portfolio/src/main/webapp/index.html

```diff
@@ -45,7 +45,16 @@ <h3>Projects I have worked on</h3>
 
       <p style="align: middle">Click <a href="images.html">here</a> to see more pictures of Sibing!</p>
       <p style="align: middle">Click <a href="blog.html">here</a> to see more hidden hobbies of Sibing!</p>
-    
+
+      <form action="/data" method="post">
+        <label>Leave Your Comment Below</label><br>
```

Comment:
Now that we have more and more strings, maybe at some point we should see how we isolate these strings so that we can support multi languages in the future.